apps/docs/src/app/platform/component-docs/platform-forms/radio-group/platform-radio-group-docs.component.html

```diff
@@ -21,24 +23,25 @@
 <code-example [exampleFiles]="listRadioGroup"></code-example>
 
 <fd-docs-section-title [id]="'def'" [componentName]="'Radio Button Group'">
-    Examples for Radio Button Group with List of objects (SelectItem Object)
+    Radio Button Group Creates Platform Radio Buttons With List of SelectItem Object. <br />
 </fd-docs-section-title>
 
 <description>
-    This example shows Regular Radio Button Group implementaion with list of Objects of SelectItem.<br />
-    Radio buttons are created with these values.
+    Creating Platform radio buttons with list of SelectItem Object.<br />
```

Comment:
same as above, name it. as 'Radio Buttons' instead

Reply:
change Done.